city_scrapers/spiders/il_lottery.py

```diff
@@ -0,0 +1,189 @@
+import re
+from datetime import datetime, time
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlLotterySpider(CityScrapersSpider):
+    name = "il_lottery"
+    agency = "Illinois Lottery Control Board"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.illinoislottery.com"]
+    start_urls = ["https://www.illinoislottery.com/illinois-lottery/lottery-control-board/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        if re.search(r'122 South Michigan Avenue, 19th Floor', response.text) is None:
```

Comment:
You should be able to do this search without regex, so it can just be

```python
if "122 South Michigan Avenue" not in response.text:
    ...
```

Reply:
Done!